app/templates/user.html

```diff
@@ -112,7 +112,7 @@ <h1 class="user-header">
             {% if followers > 0 %}
             <div class="activity activity-followers">
                 <i class="fa-solid fa-user" style="color: #76777b;"></i>
-                <p>{{ "{:,}".format(followers) }} followers</p>
+                <p>{{ "{:,}".format(followers) }} follower{{ 's' if followers != 1 }} </p>
```

Comment:
```suggestion
                <p>{{ "{:,}".format(followers) }} follower{{ 's' if followers != 1 else '' }} </p>
```

I think python won't like an inline if-statement without an else. Correct me wrong though.

Reply:
must be fine then :clueless: